### ISSUE TITLE
Transaction Problem. 

### ISSUE THREAD

**Issue by KayleyWong** [bitpay/insight#942]
Hi guys, I really need your help on this. It has stated in my wallet that i've received my bitcoins of 1.182691BTC on 29/05/18. This's my wallet address: muFjdWSy2DKStSP2gUH2aHZq6kRBva8ZDV it stated on the insight.bitpay.com. It stated confirmed. However, i can't send out to another wallet address as it stated insufficient funds. Please advise me what should i do? Is it a scam? 

My transaction ID is b2898b8c5f5f01f573e768ec21fc4f15c1e433e7255960fb71551f06cec36651. It seems that my transaction confirmed. What should I do next? I’m stuck.
[Bitcoin Address muFjdWS... _ Insight.pdf](https://github.com/bitpay/insight/files/2060792/Bitcoin.Address.muFjdWS._.Insight.pdf)

**Comment by yuntai** [comment]
https://test-insight.bitpay.com/ should show a big "TEST NET" label with warning messages.

**Comment by KayleyWong** [comment]
Hi Patrick, I bought the bitcoins from localbitcoins. And I send this bitcoins over $2k SGD to the miner where he sent me the address to be send the bitcoins too.

**Comment by BPatrick92** [comment]
Kayley, I am sorry to tell you this but the trader/miner that you bought the bitcoins from scammed you. What is his username/profile URL? Your only hope is to contact localbitcoins.com and attempt to dispute the trade.

**Comment by KayleyWong** [comment]
Patrick, I purchased my bitcoins from localbitcoins Cryptotil (https://localbitcoins.com/accounts/profile/cryptotil/) & Getbelu (https://localbitcoins.com/accounts/profile/getbelu/) And the bitcoins did sent to me to my localbitcoins wallet. From there, I send my purchased bitcoins to my coinbase wallet. The “Miner” gave me a wallet address to sent all this bitcoins over to him so he could help me on mining. This is the “miner” (https://m.facebook.com/blessing.ezelie?__xts__%5B0%5D=12.%7B%22unit_id_click_type%22%3A%22graph_search_results_item_in_module_tapped%22%2C%22click_type%22%3A%22result%22%2C%22module_id%22%3A0%2C%22result_id%22%3A100016657001547%2C%22session_id%22%3A%22e1ccc720b1c327d9ca4cf660086356f7%22%2C%22module_role%22%3A%22NONE%22%2C%22unit_id%22%3A%22browse_rl%3A15d74bb5-0568-df16-b8e5-1de04adca8a8%22%2C%22browse_result_type%22%3A%22browse_type_user%22%2C%22unit_id_result_id%22%3A100016657001547%2C%22module_result_position%22%3A0%7D) that scammed by $2k plus SGD, and blocked me on Facebook when I decided to publish him out everywhere. 

Does it help to get back my money? I’m very upset about it as I just want to earned and help my mum for her weekly injection that costs a bomb.

**Comment by BPatrick92** [comment]
No, this doesn't help sorry. Once someone has your bitcoins there is no way to reverse the transaction. I am still confused as to why you posted a testnet transaction. If you sent your money to him through coinbase I don't see how you would get a testnet transaction at the end. I am really willing to help, however, beyond this point there's nothing that can be done from a technical side. I am really sorry that you got scammed. I personally understand what it feels like and it's not a good feeling. I took at look at his Facebook profile, it looks like he's living in the US; perhaps you could threaten to contact the police as this is blatant theft.

**Comment by KayleyWong** [comment]
Patrick, the “miner” told me to create a BitPay account & wallet as the earnings is a bit to large. This is what he said, “This time it would be sent to you through bitpay then you send out to your coinbase or I will show you where to sell it and get cash straight to your bank. Now because this amount is bit large the use of bitcoin payment is the only option and there are recent rules against money laundering using crypto currency so hence they have devised a means to pay a fee to aviod this rule
But first they want you to be assured before you pay this fee that you are going to earn, so they have come up with a new system 

Go and download bitpay app on your phone
Create a wallet open it and click on “CREATE PERSONAL WALLET” on the next page enter mining as the name and BITCOIN as the coin then hit the advance options and tick testnet and single address, then create the wallet, look for recieved and then copy out the address send it to me and I will give them, they will send the bitcoins to you, you will receive them on that wallet but you will not be able to use them because they are being held in escrow by the bitpay servers, the company told bitpay servers to hold it till you pay the money laundering fee which is $1000, 
So once you pay it, they will ask bitpay to release it so you can use it, after that then you can send it to coinbase or any wallet and cash out to your bank account, it’s simple 
Sorry about the fee you have to pay it’s a new development but atleast you are assured of your earnings
I was just informed of this now.”

**Comment by Mickieki** [comment]
rukmeay@hotmail.com

**Comment by BPatrick92** [comment]
Sorry, Kayley. There's nothing that can be done. You got scammed. I am sorry that this happened to you. I would try to contact the proper authorities.

**Comment by KayleyWong** [comment]
Really very upset. 😭thank you Patrick for the help.

**Comment by dannylee71** [comment]
My wallet key:3BoFcMLSJDH5oc3gPeE4F8JuqTAmvA7B4
You can withdraw in my account!
Thank you

2018年6月9日 下午5:43，"KayleyWong" <notifications@github.com>写道：

Hi Patrick, I bought the bitcoins from localbitcoins. And I send this
bitcoins over $2k SGD to the miner where he sent me the address to be send
the bitcoins too.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/bitpay/insight/issues/942#issuecomment-395955615>, or mute
the thread

**Comment by KayleyWong** [comment]
Danny. what do you mean?

**Comment by Mickieki** [comment]
You can pay them right from this email.


Google Pay

You received a money request from MY MiCkY



$ 200 .





Send


Decline








img

MY MiCkY

rukmeay@gmail.com





Time

Sep 04, 2018 11:50 AM PDT





Method







ID

L.1B98-DF27-9601-005C





Google





Google Play App Store





You can also try new features at pay.google.com

How to recognize suspicious emails | Terms of Service
Help Center | Privacy Notice

Funds subject to processing by Google Payment Corp. Google receives  
consumer report info to protect you from fraud and inform transaction  
decisions.
